BridgeAppSDK/DataArchiving/SBADataArchive.m

```diff
@@ -163,6 +163,12 @@ - (void)insertDictionaryIntoArchive:(NSDictionary *)dictionary filename:(NSStrin
 
 - (void)insertDataIntoArchive :(NSData *)data filename: (NSString *)filename
 {
+    // Check that the file has not already been added
```

Comment:
This is a belt and suspenders check. There were two different bugs that were causing the issue and they have been resolved in a way that is appropriate to the error condition. In this case, it is saying "in a production app, pass the data in such a way that it can be collected by the service. in debug, throw an assert."